app/controllers/claims/support/schools_controller.rb

```diff
@@ -1,2 +1,9 @@
 class Claims::Support::SchoolsController < Claims::Support::ApplicationController
+  def index
+    @schools = Claims::School.includes(:gias_school).order("gias_schools.name")
+  end
+
+  def show
+    @school = Claims::School.find(params[:id])
```

Comment:
Can we do `params.require(:id)`? Just gives us more information if it goes wrong and it also acts like a validation